elastic/kibana
# [Security solution] Risk score platinum CTA

## Summary


Sets a call to action on the Entity Analytics page for users with a Basic license to upgrade to Platinum. 

![paywall](https://user-images.githubusercontent.com/6935300/190519243-779f9d74-75cd-49d8-868d-ff2a0f935135.gif)

### Testing instructions
1. Use [dev tools to downgrade](https://p.elstc.co/paste/eLqGlaMC#ZUF8T46lvYozsypdoAsDnBkWH+KIWxDF+YdEzLy0wfX) license to basic
2. check that paywall appears on entity analytics page
3. check that Upgrade button links to license management area of stack managemebnt
4. Use [dev tools to upgrade](https://p.elstc.co/paste/Ektxe5qd#gihIOvkriE8ztHh9OojxLH77ai4M3tak7IO7bGV0nXq) license back to platinum
5. check that entity analytics page is working woohoo
